packages/mui-material/src/styles/responsiveFontSizes.spec.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
```

Comment:
I think this test should be in `test/typescript/moduleAugmentation`, otherwise it will affect the project.

Reply:
@siriwatknp Done